pymeasure/instruments/hp/hp856Xx.py

```diff
@@ -1392,131 +1609,1374 @@ def marker_minimum(self):
-        self.frequency_offset_values = [0, self.MAX_FREQUENCY]
-        self.marker_frequency_values = [0, self.MAX_FREQUENCY]
+            instr.peak_excursion = 2
+            if instr.peak_excursion == 2:
+                pass
 
-    mixer_mode = Instrument.control(
-        "MXRMODE?", "MXRMODE %s",
-        """
-        Specifie the mixer mode. Select either the internal mixer
-        or supply an external mixer. Takes enum 'MixerMode' or string 'INT', 'EXT'
         """,
-        validator=strict_discrete_set,
-        values=[e for e in MixerMode]
+        validator=strict_range,
+        values=[0.1, 99]
     )
 
-    conversion_loss = Instrument.control(
-        "CNVLOSS?", "CNVLOSS %s",
+    def marker_to_reference_level(self):
+        """Set the reference level to the amplitude of an active marker.
+
+        If no marker is active, 'marker_to_reference_level' places a
+        marker at the center of the trace and uses that marker amplitude
+        to set the reference level.
         """
-        Compensate for losses outside the instrument when in external
-        mixer mode (such as losses within connector cables, external mixers, etc.).
-        'conversion_loss' specifies the mean conversion loss for the current harmonic band.
-        In a full frequency band (such as band K), the mean conversion loss is defined as the
-        minimum loss plus the maximum loss for that band divided by two.
-        Adjusting for conversion loss allows the system to remain
-        calibrated (that is, the displayed amplitude values have the conversion loss incorporated
-        into them). The default value for any band is 30 dB. The spectrum analyzer must be in
-        external-mixer mode in order for this command to work. When in internal-mixer mode,
-        querying 'conversion_loss' returns a zero.
-        """,
-        validator=strict_range,
-        values=[0, float("inf")]
-    )
+        self.write("MKRL")
 
-    def fullband(self, band):
+    def marker_delta_to_span(self):
+        """Set the frequency span equal to the frequency difference between two
+        markers on a trace.
+
+        The start frequency is set equal to the frequency of the left-
+        most marker and the stop frequency is set equal to the frequency
+        of the right-most marker.
         """
-        Select a commonly-used, external-mixer frequency band, as shown
-        in the table. The harmonic lock function :attr:`harmonic_number_lock` is also set; this
-        locks the harmonic of the chosen band. External-mixing functions are not available with
-        an HP 8560A Option 002. Takes frequency band letter as string.
+        self.write("MKSP")
 
-        .. list-table:: Title
-            :widths: 25 25 25 25
-            :header-rows: 1
+    def marker_to_center_frequency_step_size(self):
+        """Set the center frequency step-size equal to the frequency value of
+        the active marker."""
+        self.write("MKSS")
```

Comment:
This is an action (no parameter and no return value), please reformulate the name with a verb, for example `set_marker_to...`.